google/cloud/videointelligence_v1beta2/gapic/video_intelligence_service_client.py

```diff
@@ -191,9 +191,9 @@ def __init__(
     # Service calls
     def annotate_video(
         self,
+        features,
```

Comment:
Replacement needs to be updated to include v1beta2 (or maybe just take this, since this is in the beta?)

https://github.com/googleapis/python-videointelligence/blob/ce1e7defb7597f5afd3eb22b23259407382e5faa/synth.py#L66-L80